src/Servers/Kestrel/README.md

```diff
@@ -1,8 +1,39 @@
-KestrelHttpServer
-=================
+# Kestrel
 
-Kestrel is a cross-platform web server for ASP.NET Core.
+Kestrel is our cross-platform web server that is included and enabled by default in ASP.NET Core.
 
-## File logging for functional test
+## Description
 
-Turn on file logging for Kestrel functional tests by specifying the environment variable ASPNETCORE_TEST_LOG_DIR to the log output directory.
+The following contains a description of the sub-directories.
```

Comment:
I think we should also add something for the end-to-end tests in https://github.com/dotnet/aspnetcore/tree/main/src/Servers/Kestrel/test.